person.rb

```diff
@@ -1,19 +1,22 @@
 require_relative 'nameable'
 require_relative 'capitalize_decorator'
 require_relative 'trimmer_decorator'
+require_relative 'rental'
+require_relative 'book'
 
 class Person
+  attr_reader :id, :rentals
```

Comment:
- [ ] You are supposed to have Getters and Setters for @rentals
`attr_accessor :name, :age, :rentals`
The getter method allows you to access the value of the @rentals instance variable from outside the class, while the setter method allows you to assign a new value to the @rentals instance variable.

Reply:
Thanks, done.